cves/kernel/CVE-2018-13100.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: In the implementation of the F2FS file system within the Linux kernel, an issue arose due to improper validation of metadata when mounting an image. This could lead to a divide-by-zero error when dealing with a corrupted or malicious file system image, leading to a system crash or possibly out of bounds memory access and privilege escalation in some cases.
```

Comment:
Would you be to explain in a little bit more detail about what metadata is effected when mounting an image?